tests/test_connector.rs

```diff
@@ -457,12 +458,23 @@ fn test_deposit_with_same_proof() {
         p.assert_success()
     }
     let promises = call_deposit_near(&contract, CONTRACT_ACC);
-    let l = promises.len();
-    let p = promises[l - 2].clone();
-    match p.unwrap().status() {
-        ExecutionStatus::Failure(_) => {}
-        _ => panic!(),
-    }
+    let promise = &promises[promises.len() - 2];
+    check_execution_status_failure(
+        promise.as_ref().unwrap().outcome().clone().status,
+        "ERR_PROOF_EXIST",
```

Comment:
As we have a constant defined for `ERR_NOT_ENOUGH_BALANCE_FOR_FEE`, please add another constant for this error as well.

Reply:
I think we don't need constants for one-time-used values.

Reply:
I suggested this for consistency. But I'm ok if we keep it like this, as this is quite minor.